scripts/integrations/angular/src/app/events/events.component.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { Component } from '@angular/core';
+
+interface ComponentEvent {
+	name: string;
+	counter: number;
+	message: string;
+}
+
+@Component( {
+	selector: 'app-events',
+	templateUrl: './events.component.html'
+} )
+export class EventsComponent {
+	public isReadOnly = false;
+
+	isHidden = false;
+
+	isRemoved = false;
+
+	inline = false;
+
+	public componentEvents: ComponentEvent[] = [];
+
+	clearEventsLog() {
```

Comment:
Also, this method is inconsistent with the next ones in the return type annotation.